package-lock.json

```diff
@@ -1,10066 +1,8 @@
 {
   "name": "pwa-kit",
-  "version": "2.7.3-preview",
-  "lockfileVersion": 2,
+  "version": "2.7.3-preview.2",
+  "lockfileVersion": 1,
```

Comment:
I'm late but this file is using the wrong lockfileVersion. 

It should keep using `"lockfileVersion": 2,`